pkgs/os-specific/linux/uhk-udev-rules/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, stdenv, uhk-agent, fetchurl }:
+
+stdenv.mkDerivation {
+  pname = "uhk-udev-rules";
+  version = lib.getVersion uhk-agent;
+  inherit (uhk-agent);
+  dontBuild = true;
+  dontUnpack = true;
+  installPhase = ''
+     install -D -m 644 ${uhk-agent.out}/rules/50-uhk60.rules $out/lib/udev/rules.d/50-uhk60.rules
+  '';
+
+  meta = {
+    description = ''
+    udev rules for UHK keyboards
+    from https://ultimatehackingkeyboard.com.
+    Users must be in the input group. E.g. \n
+    users.extraGroups.input.members = [ "userName" ];\n
```

Comment:
```suggestion
```
I don't think that's the right place to put it. It also doesn't need to explain how to configure groups. (besides, extraGroups has been renamed to groups and some administrators such as myself prefer `users.users.<name>.extraGroups`)